filebeat/tests/system/test_modules.py

```diff
@@ -140,7 +140,7 @@ def run_on_file(self, module, fileset, test_file, cfgfile):
         output.write(" ".join(cmd) + "\n")
 
         local_env = os.environ.copy()
-        local_env["TZ"] = 'Etc/UTC'
```

Comment:
This is the only change in code, the rest are the result when regenerating golden files.